Source/WebInspectorUI/UserInterface/Views/Variables.css

```diff
@@ -24,265 +25,150 @@
  */
 
 :root {
-    --z-index-highlight: 64;
```

Comment:
it seems like a lot more than color CSS variables were modified in this PR?  why?  perhaps a bad rebase?

Reply:
I am alphabetizing them for better readability, I can do it in a separate pr.

Reply:
im not sure alphabetization will help with readability in this case.  sure it could be organized a bit better, but i think having things be grouped into related concepts (e.g. `--background-color` should probably be right before `--selected-background-color`)

but yes regardless that should absolutely be a separate PR